public/apps/account/password-reset-panel.tsx

```diff
@@ -19,6 +19,9 @@ import {
   EuiButtonEmpty,
   EuiCallOut,
   EuiFieldPassword,
+  EuiFlexGroup,
```

Comment:
Should we be using the OUI component library?

Reply:
There is actually a deep link from Eui to Oui - see here: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/package.json#L139, I guess this is due to legacy reasons and refactoring may be a pain.